packages/babel-plugin-proposal-class-properties/test/fixtures/nested-class/super-call-in-key/output.js

```diff
@@ -23,8 +23,10 @@ function (_Hello) {
     var _this2 = _this = babelHelpers.possibleConstructorReturn(this, babelHelpers.getPrototypeOf(Outer).call(this));
 
     let Inner = function Inner() {
+      var _this3 = this;
```

Comment:
Can we somehow avoid always assigning `this`?

Reply:
thanks for your suggestion, theses assigning has cleanup if REFs less than one.